vnext/Desktop/DevSupportManager.cpp

```diff
@@ -59,27 +59,33 @@ string DevSupportManager::GetJavaScriptFromServer(
   try {
     Url url(bundleUrl);
     auto const resolveResult = m_resolver.resolve(url.host, url.port);
-    tcp::socket socket{m_context};
-    connect(socket, resolveResult);
+    boost::beast::tcp_stream stream{m_context};
+
+    stream.connect(resolveResult);
 
     request<string_body> request{verb::get, url.Target(), 11};
     request.set(field::host, url.host);
     request.set(field::user_agent, BOOST_BEAST_VERSION_STRING);
 
-    write(socket, request);
+    write(stream, request);
 
     flat_buffer buffer;
     response<string_body> response;
 
     parser<false, string_body> p{std::move(response)};
     p.body_limit(25 * 1024 * 1024); // 25MB (boost default of 1MB is too small for dev bundles)
 
-    read(socket, buffer, p.base());
+    read(stream, buffer, p);
     response = p.release();
     std::stringstream jsStringStream;
     jsStringStream << response.body();
     // TODO: Check if UTF-8 processing is required.
 
+    boost::system::error_code ec;
+    stream.socket().shutdown(tcp::socket::shutdown_both, ec);
+    if (ec && ec != boost::beast::errc::not_connected)
```

Comment:
Couple questions here:
1. What happens if we just let the stream dtor run
2. There's an overload of shutdown() that throws instead of returning an error_code. If we want to filter out errors, I can understand why you'd want to use the error_code version, but why do we expect this error in particular?

Reply:
Looks like we use this pattern in a number of places. I wonder if it might make sense to put it in a helper somewhere.

Reply:
It's possible the behavior would be correct if we let the dtor run.
I decided to be explicit, because the provided samples also make the shutdown this way.

I'll take a look at the throwing variant of `shutdown`.

Reply:
The destructor of `boost::asio::basic_stream_socket` does not seem to carry this operation implicitly.